slurm/merge.jl

```diff
@@ -30,17 +33,99 @@ function main(config::Dict)
         # Save dataset to disk
         get!(D, "meta", Dict{String,Any}())
         D["meta"]["config"] = config_str
-        OPFGenerator.save_h5(joinpath(export_dir, "$(casename)_$(dataset_name).h5"), D);
+
+        if dataset_name == "input"
+            OPFGenerator.save_h5(joinpath(export_dir, "input.h5"), D)
+        else
+            mkpath(joinpath(export_dir, dataset_name))
+            OPFGenerator.save_h5(joinpath(export_dir, dataset_name, "meta.h5"), D["meta"])
+            OPFGenerator.save_h5(joinpath(export_dir, dataset_name, "primal.h5"), D["primal"])
+            OPFGenerator.save_h5(joinpath(export_dir, dataset_name, "dual.h5"), D["dual"])
+        end
 
         GC.gc()
     end
 
-    return nothing    
+    rm(joinpath(export_dir, "res_h5"), recursive=true)
+
+    rng = MersenneTwister(42)
+
+    feasible = nothing
+    n_samples = nothing
+
+    # find inputs that are feasible for all OPFs
+    for opf in OPFs
+        h5open(joinpath(export_dir, opf, "meta.h5"), "r") do f
+            if isnothing(feasible)
+                n_samples = length(read(f["termination_status"]))
+                feasible = trues(n_samples)
+            end
+            feasible .&= (String.(read(f["termination_status"])) .== "OPTIMAL") .| (String.(read(f["termination_status"])) .== "LOCALLY_SOLVED")
+            feasible .&= (String.(read(f["primal_status"])) .== "FEASIBLE_POINT")
+            feasible .&= (String.(read(f["dual_status"])) .== "FEASIBLE_POINT")
```

Comment:
This isn't optimal, but it does the trick

Reply:
do we need the `String.`? don't we always export those as strings?

Reply:
I think you're right, let me check

Reply:
Yep, it's redundant. I'll remove it